modules/Utilities/include/Utils.hpp

```diff
@@ -182,6 +204,17 @@ namespace YarpHelper
      */
     bool yarpListToiDynTreeVectorDynSize(const yarp::os::Value& input, iDynTree::VectorDynSize& output);
 
+    /**
+     * Extract an iDynTree VectorDynSize from searchable
+     * @param config is the searchable object;
+     * @param key the name to check for;
+     * @param vector the iDynTree vector.
+     * @return true/false in case of success/failure
+     */
+    bool getiDynTreeVectorDynSizeFromSearchable(const yarp::os::Searchable& config,
```

Comment:
Maybe you can write a single method ``getVectorFromSearchable`` where the type of vector is a template. In this way you would have a single version of this method instead of 3.

Reply:
Done in 2534dde